contracts/cw3-fixed-multisig/src/contract.rs

```diff
@@ -238,12 +245,16 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
         QueryMsg::Proposal { proposal_id } => to_binary(&query_proposal(deps, env, proposal_id)?),
         QueryMsg::Vote { proposal_id, voter } => to_binary(&query_vote(deps, proposal_id, voter)?),
         QueryMsg::ListProposals { start_after, limit } => {
-            to_binary(&list_proposals(deps, env, start_after, limit)?)
+            let reverse = false;
+            to_binary(&list_proposals(deps, env, reverse, start_after, limit)?)
         }
         QueryMsg::ReverseProposals {
             start_before,
             limit,
-        } => to_binary(&reverse_proposals(deps, env, start_before, limit)?),
+        } => {
+            let reverse = true;
+            to_binary(&list_proposals(deps, env, reverse, start_before, limit)?)
+        }
```

Comment:
do we need both of these endpoints, or can we get away with just `QueryMsg::ListProposals` with an optional `reverse` field?